### PR TITLE
Adding information on kill signals

### PR DESCRIPTION
**Pull request by eirikurj**
## Purpose
Adding basic information on kill signals that ADflow support and some basic usage.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### CODE REVIEW

**Review by marcomangano — COMMENTED** [review]
Oh man if I knew this before I could have saved hours of my life

File: doc/tutorial.rst [hunk 1]
```diff
@@ -201,3 +201,24 @@ There are two common cases. The span of the wing is in, y direction (rotation ab
...
+
+The definition of an iteration is different for steady and unsteady. For steady it means after the current iteration, for unsteady after the current time step.
+
+The signals are enabled by default but can be switched off or disabled at compile time using the compiler flag ``-DUSE_NO_SIGNALS``.
+
+To use the signals from the command line run::
+
+  kill -USR1 <PID>
+
+where ``<PID>`` is the process id of the mpi process. These signals are often used when debugging. For instance, the ``-USR1`` signal can be useful to write out a semi-converged solution for further investigation, and the ``-USR2`` can be used to stop a stalled solution without loss of data. Other use-cases are also possible.
```
Comment (marcomangano) [inline doc/tutorial.rst:219]: Could you clarify how to find the `<PID>` on both our local machines and HPC?

Comment (sseraj) [reply doc/tutorial.rst:219]: I assume you can use `top` on a compute node the same as locally?

**Review by sseraj — APPROVED** [review]
I checked that both of these kill signals work as described. This is very useful!

File: doc/tutorial.rst (hunk 1 above)
Comment (eirikurj) [reply doc/tutorial.rst]: Yes, one could use `top`, or `ps -ef` for example. Added that to the docs.

**Review by sseraj — APPROVED** [review]

File: doc/tutorial.rst (hunk 1 above)
Comment (nwu63) [reply doc/tutorial.rst]: Yeah I mentioned this in the HPC guide -- you can just ssh into a compute node that's running your job, and monitor/send signals as you wish.

**Review by nwu63 — APPROVED** [review]

**Pull request merged**